doc/project/testing.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: doc
+title: Testing
+---
+
+In this section we will discuss how to test Scala.js code managed as [Full cross project](./cross-build.html). All names for example subproject name such as `fooJVM` are taken from [cross compile example](https://github.com/scala-js/scalajs-cross-compile-example).
+
+
+## Directory Structure
+
+    <project root>
+     +- jvm
+     |   +- src/main/scala
+     |   +- src/test/scala
+     +- js
+     |   +- src/main/scala
+     |   +- src/test/scala
+     +- shared
+         +- src/main/scala
+         +- src/test/scala
+
+Similarly to `{shared|js|jvm}/src/main/scala` folders that contain your code, there can be also `{shared|js|jvm}/src/test/scala` folders that would contain your tests. Tests that verify correctness of shared code should go to `shared` test folder, while tests that check Scala JVM code or Scala.js code only should go to `jvm` or `js` test folders respectively. When you call `sbt> test` two iterations of tests will run:
+
++ Iteration 1: all tests residing in `shared` and `jvm` will be executed testing Scala JVM code.
++ Iteration 2: all tests residing in `shared` and `js` will be executed testing Scala.js code.
+
+If you are interested in running only one of those iterations you could call `sbt> fooJVM/test` or `sbt> fooJS/test` accordingly. 
```

Comment:
I would *first* present the logic to test separately. Then just remind that with an aggregate project, calling `test` will just run `fooJVM/test` and `fooJS/test`.